.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
## OPTIONAL
- Please don't ignore `dist`. kindly send it out of gitignore because the dist folder is the build folder which contains all the files and folders which can be hosted on the server